highfive/newpr.py

```diff
@@ -102,6 +106,7 @@ def api_req(self, method, url, data=None, media_type=None):
         body = f.read()
         return { "header": header, "body": body }
 
+    @retry(HttpClientError, delay=1, backoff=2, max_delay=3)
```

Comment:
Just so I am clear from the documentation of the [retry package](https://pypi.org/project/retry/), this will allow up to three attempts: the initial attempt, another one second after that fails, and another two seconds after the second attempt fails.

It would likely be easier for others to read if we just used the `tries` argument and drop `max_delay`.